src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -119,17 +136,22 @@ static void _pxp_interrupt_init(void)
     PXP_ASSERT_MSG(xPXPIdleSemaphore, "xSemaphoreCreateBinary failed!");
 
     NVIC_SetPriority(PXP_IRQ_ID, configLIBRARY_MAX_SYSCALL_INTERRUPT_PRIORITY + 1);
```

Comment:
With PXP=1 and OS_NONE the build fails on MCUXpresso SDK projects. This is a common build configuration for us. The build fails because SDK_OS_FREE_RTOS is defined (by MCUX) but configLIBRARY_MAX_SYSCALL_INTERRUPT_PRIORITY is no longer found.

Since you similarly need CONFIG_LV_Z_PXP_INTERRUPT_PRIORITY for __ZEPHYR__ we could probably use a common abstraction for interrupt priory that will resolve the build for both cases. What do you think

Reply:
hi @faxe1008 
Can you please address this issue? Try to not mix anylonger the `SDK_OS_FREE_RTOS` /`__ZEPHYR__` with `LV_USE_OS`.
We can fix by ourself the other problem with multithreading mentioned [here](https://github.com/lvgl/lvgl/pull/6159#discussion_r1593353421) but please fixup this PR before merging.

Reply:
@nicusorcitu I am not quite sure if I understood everything correctly but I ammended the commit with a change that hopefully fixes this.

Reply:
Actually it makes it worst because now there is no distinction at all between FreeRTOS and Bare Metal for PXP.

Please revert your last changes.
What I men was to have kind of an abstraction os function like "lv_os_interrupt_enable()" or "lv_os_interrupt_disable()" but I figure out that this is basically something platform dependent and not OS dependent.

To keep things simple and conclude on this PR:
1. please revert your last changes from [here](https://github.com/lvgl/lvgl/compare/fd1e2466a5133625483e819ee3bf08738d80b9f2..158a93acf415225cac7c74f0d1a0536a76b793fa)
2. add back the inclusion necessary for freertos irq priority (for freertos)
```
#if defined(SDK_OS_FREE_RTOS)
    #include "FreeRTOS.h"
#endif
```

Reply:
Performed the suggested changes :^)

Reply:
I do not see them. Can you please point a link?

Reply:
I forgot to push the changes and was out of office for the week. Pushed it right now :^)